lib/fluent/plugin/in_rds_slowlog.rb

```diff
@@ -13,9 +13,34 @@ class Fluent::Plugin::Rds_SlowlogInput < Fluent::Plugin::Input
   config_param :password,     :string,  :default => nil, :secret => true
   config_param :interval,     :integer, :default => 10
   config_param :backup_table, :string,  :default => nil
+  # The encoding after conversion of the input.
```

Comment:
This is just a my thought. Feel free to ignore it.
We can describe parameters how to work with `desc` method which can display usage with `fluent-plugin-config-format`.
So, we might have to add parameter descriptions with `desc` method.
But this is not issue, just a suggestion.